server/models.py

```diff
@@ -23,6 +31,25 @@ def full_name(self):
             return self.last_name
         return "%s %s" % (self.first_name, self.last_name)
 
+    @property
+    def is_authenticated(self):
+        return True
+
+    @property
+    def is_active(self):
+        return self.active
+
+    @property
+    def is_admin(self):
+        return self.admin
+
+    @property
+    def is_anonymous(self):
+        return False
+
+    def get_id(self):
+        return str(self.id)
```

Comment:
What are `is_authenticated`, `is_admin`, `is_anonymous` and `get_id` for? I don't see that they are used, are some other modules using them?